baselines/teams.md

```diff
@@ -512,45 +559,55 @@ To configure the Meeting policies for cloud video recording:
 
 5.  Select **Save**.
 
+
 If there is a legitimate business need, *specific* users can be given
 permission to record meetings. To allow specific users the ability to
 record meetings:
 
 1.  Sign in to the **[Microsoft Teams admin
     center](https://admin.teams.microsoft.com).**
 
-2.  Select **Meetings** -\> **Meeting policies**.
+2.  Select **Meetings** > **Meeting policies**.
 
 3.  Create a new policy by selecting **Add**. Give this new policy a
     name and appropriate description.
 
 4.  Under the **Recording & transcription** section, set **Cloud
     recording** to **On**.
 
-5.  Under the **Recording & transcription** section, set **Store
-    recordings outside of your country or region** to **Off**.
-
-6.  Select **Save**.
+5.  Select **Save**.
 
-7.  After selecting **Save**, a table displays the set of policies.
+6.  After selecting **Save**, a table displays the set of policies.
     Select the row containing the new policy, then select **Manage
     users**.
 
-8.  Assign the users that need the ability to record to this policy.
+7.  Assign the users that need the ability to record to this policy.
 
-9.  Select **Apply**.
+8.  Select **Apply**.
 
-## 10. Recording of Live Events
+#### MS.TEAMS.9.2v1, instructions:
 
-Live events are recorded by default. Agencies should increase their
-privacy by changing the policy so that events are only recorded at the
-organizer’s discretion.
+1.  Sign in to the **[Microsoft Teams admin
+    center](https://admin.teams.microsoft.com).**
+
+2.  Select **Meetings** > **Meeting policies**.
+
+3.  Select the **Global (Org-wide default)** policy.
+   
+4.  Under the **Recording & transcription** section, set **Store
+    recordings outside of your country or region** to **Off**.
+
+5.  Select **Save**.
+
+## 10. Recording of Live Events
+This section helps reduce security risks related to the recording user permissions for Live Events.
 
 ### Policies
 #### MS.TEAMS.10.1v1
-Record an event SHOULD be set to Organizer can record.
-- _Rationale:_ TODO
+Record an event SHOULD be set to Organizer can record Global (org-wide default) meeting policy and all custom policies.
+- _Rationale:_ The security risk of the default settings for Live Events are that Live Events are avaialble to be recoreded by all participants by default. By limiting the recording permissions to only the organizer this minimizes the security risk to the organizer's discretion for these Live Events.
```

Comment:
Minor typos
```suggestion
- _Rationale:_ The security risk of the default settings for Live Events is that Live Events are available to be recorded by all participants by default. By limiting the recording permissions to only the organizer this minimizes the security risk to the organizer's discretion for these Live Events.
```

Also, I get what we're saying here, but the justification here feels weak to me. What is the security risk of having a live event recorded? Assuming proper SharePoint controls are in place, the recordings should be safe. Plus these live events things tend to be intended for large audiences, such as a town hall meeting. Meaning you wouldn't expect them to have info that isn't appropriate for the org at-large in the first place. Have we considered cutting this control? I think I remember some agencies expressing similar feelings during the pilot meetings, but not entirely sure.